### ISSUE TITLE
[FEATURE REQUEST] Improve the contrast of game titles in search results

### ISSUE THREAD

**Issue by Korb** [tfedor/AugmentedSteam#2268]
### Current Behavior (if any)

![Image](https://github.com/user-attachments/assets/33213fef-e07f-4deb-af3b-a43e2515348b)

| Non-ignored | Ignored |
| - | - |
| Foreground: `#C6D4DF` | Foreground: `#313C48` |
| Background: `#16202D` | Background: `#192635` |
| Contrast ratio: **10.13:1** | Contrast ratio: **1.365:1** |
| 1.4.3 Contrast (Minimum) (AA) | 1.4.3 Contrast (Minimum) (AA) |
| ✅ Pass for large and regular text | ❌ Fail for large and regular text |
| 1.4.6 Contrast (Enhanced) (AAA) | 1.4.6 Contrast (Enhanced) (AAA) |
| ✅ Pass for large and regular text | ❌ Fail for large and regular text |

### Desired Behavior

| Non-ignored | Ignored |
| - | - |
| Foreground: `#CBD8E2`…`#EFF3F6` | Foreground: `#A4B2C1` |
| Contrast ratio: **10.555:1**…**13.7:1** | Background: `#192635` |
|  | Contrast ratio: **7.085:1** |
|  | 1.4.3 Contrast (Minimum) (AA) |
|  | ✅ Pass for large and regular text |
|  | 1.4.6 Contrast (Enhanced) (AAA) |
|  | ✅ Pass for large and regular text |

### Motivation / Use Case for Changing the Behavior

[a11y] WCAG 2.1/2.2[^1]
[^1]: https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html
